lib/util/params.c

```diff
@@ -21,6 +21,7 @@
 
 #define BUFSIZE 30
 #define FIRST_PRINTABLE 65 /* ord('A') = 65 */
+#define VERSION_SHORT_OPT 0
```

Comment:
Why not just use 'V'?

Reply:
I decided that -V is not really a convention, and it's used by a lot of other command-line tools for other purposes. I did not want to restrict tools from using -V, as -v is already taken.

I did think of using -V and then check to see if the user options include a -V and then fall back to no short option. But I thought it was not worth the complexity.

Reply:
Nah, better to be consistent. I can live with no short option for `--version` :)